### DISCUSSION TITLE
Any way to get this working with RecordRTC?

### DISCUSSION THREAD

**Discussion by oagix** [samhirtarif/react-audio-visualize#6] [Q&A]
I know it works with the MediaRecorder, but trying to use the live component to visualize a stream or recorder built using: https://recordrtc.org/

RecordRTC has better support for older devices & browsers, but the recorder it exposes is not a MediaRecorder:

<img width="529" alt="image" src="https://github.com/samhirtarif/react-audio-visualize/assets/8274669/09b7216f-2a8f-47e5-a692-b0346db611c8">

<img width="724" alt="image" src="https://github.com/samhirtarif/react-audio-visualize/assets/8274669/14de05e8-2c98-4438-8122-ce15297f74dd">

**Comment by samhirtarif** [comment]
I'll need to have a look at this. And the recorder can be one of the many types listed in the first screenshot?

**Reply by oagix** [reply]
Yeah, what I eventually ended up doing was re-create the component for the live visualizer to make it work. 

Maybe it’s a bit of a niche thing but I think it would be useful to have native support in the library.
